packages/tokens/dist/devdot/css/helpers/color.css

```diff
@@ -1,6 +1,6 @@
 /**
  * Do not edit directly
- * Generated on Fri, 04 Mar 2022 17:16:48 GMT
+ * Generated on Wed, 09 Mar 2022 00:35:25 GMT
```

Comment:
i don't think updating the docs should also generate a new set of `dist/` changes, should it?

Reply:
@Dhaulagiri sorry, my bad! I was testing all the commands from the packages/tokens directory and shouldn't have included those files

just rebased